Formula/now-cli.rb

```diff
@@ -16,6 +16,8 @@ class NowCli < Formula
 
   depends_on "node"
 
+  deprecate! date: "2021-01-31"
```

Comment:
We're adding reasons to `deprecate!` and `disable!` (will be required by `brew style` soon)

```suggestion
  deprecate! date: "2021-01-31", because: "is deprecated upstream"
```

Or, once https://github.com/Homebrew/brew/pull/8530/files is merged and there's a new release,

```suggestion
  deprecate! date: "2021-01-31", because: : deprecated_upstream
```